### DISCUSSION TITLE
Display images as embed in private messages

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17385] [Ideas] [answered]
Same result as if I clicked the link, except without the struggle of tabbing out
As to not ping IP in case of malicious user, could always add a spoiler and *click to reveal* to load image

![image](https://user-images.githubusercontent.com/40531871/159226548-51b7f130-dc71-4fc1-b08a-00db49f0fc09.png)

Could be used by IRC bots to display pp graphs, osu skill bot to display spider chart with the skill of the user, and more

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13168
